src/aiida/cmdline/utils/decorators.py

```diff
@@ -248,7 +248,13 @@ def deprecated_command(message):
         @deprecated_command('This command has been deprecated in AiiDA v1.0, please use 'foo' instead.)
         def mycommand():
             pass
+
+    Ironicallly, this decorator itself has been deprecated. ``verdi`` commands that should be deprecated should simply
+    use the ``deprecated`` argument in the ``command`` decorator and specify the deprecation message.
```

Comment:
Maybe we put this into a sphinx deprecation decorator?
```suggestion
    .. deprecated:: 2.6
       Ironicallly, this decorator itself has been deprecated. ``verdi`` commands that should be deprecated should simply
       use the ``deprecated`` argument in the ``command`` decorator and specify the deprecation message.
```

Reply:
Done